pyproject.toml

```diff
@@ -1,5 +1,10 @@
 [build-system]
-requires = ["setuptools>=42", "wheel", "numpy", "setuptools_scm[toml]>=3.4"]
+requires = [
+    "setuptools>=42",
+    "setuptools_scm[toml]>=3.4",
+    "wheel",
+    "oldest-supported-numpy",
```

Comment:
Well that is a handy dandy feature isn't it

Reply:
We may want to do this for other `spacetelescope` projects that have C code that need `numpy` installed to build/compile.  Maybe we should file issues for the others as well.